hello/Makefile

```diff
@@ -0,0 +1,22 @@
+include ../Makefile.inc
+UPSTREAM=hello.tar.gz
```

Comment:
Would prefer to have the source files uncompressed, so people can play with them.

Reply:
Ah - ok. Okies ... will do so.